src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,43 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ imgUrl, title, imdbUrl, description }) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
```

Comment:
```suggestion
        <img src={imgUrl} alt="Film logo" />
```
If there are only 2 short attributes, they can fit on one line perfectly fine.